### PR TITLE
Remove rvm

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review when green.

Thanks.

**Comment by josephbridgwaterrowe** [comment]
Although it needs rebasing before merge.

File: Capfile [hunk 1]
```diff
@@ -7,9 +7,8 @@ require 'capistrano/deploy'
 # Include tasks from other gems included in your Gemfile
 require 'capistrano/bundler'
 require 'capistrano/rails/assets'
-require 'capistrano/rails/migrations'
+# require 'capistrano/rails/migrations'
```
Comment (michaelachrisco) [inline Capfile]: Why was this removed?

File: Gemfile [hunk 2]
```diff
@@ -27,17 +28,15 @@ gem 'therubyrhino'
 gem 'uglifier', '>= 1.0.3'
 
 group :development, :test do
-  gem 'capistrano', '~> 3.3.0'
+  gem 'capistrano'
```
Comment (michaelachrisco) [inline Gemfile]: Don't we want capistrano versions to be explicit?

File: config/deploy.example.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# config valid only for current version of Capistrano
+lock '3.3.5'
```
Comment (michaelachrisco) [inline config/deploy.example.rb]: You have locked capistrano at 3.3.5 but Capistrano version is 3.4.0.

File: Capfile [hunk 4]
```diff
@@ -7,9 +7,8 @@ require 'capistrano/deploy'
...
 require 'capistrano/rvm'
```
Comment (michaelachrisco) [inline Capfile]: Since you removed rvm, do you need this?

Comment (josephbridgwaterrowe) [inline Capfile]: I put rvm back :)

File: config/deploy.example.rb (hunk 3 above)
Comment (josephbridgwaterrowe) [inline config/deploy.example.rb]: Shit, I need to update this. :)

File: Gemfile (hunk 2 above)
Comment (josephbridgwaterrowe) [inline Gemfile]: Yes, thanks. I'll set to 3.4.0.

File: Capfile (hunk 1 above)
Comment (josephbridgwaterrowe) [inline Capfile]: It was causing issues when I was deploying, although I'm not sure if I fixed them or not.

I'll give it another run with it back in there.

Comment (josephbridgwaterrowe) [inline Capfile]: It blows up with a big smelly error. I'm going to keep it out for now.

**Comment by michaelachrisco** [comment]
Looks good. Rebase and I will merge.

**Comment by michaelachrisco** [comment]
I am going to rename this merge Update Deploy since we were not able to remove RVM.
